docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -233,408 +122,121 @@ is used.
-are under 10 megabytes in size:
-
-.. code-block:: sql
-
-    ALTER TABLE test_table EXECUTE optimize(file_size_threshold => '10MB')
-
-You can use a ``WHERE`` clause with the columns used to partition
-the table, to apply ``optimize`` only on the partition(s) corresponding
-to the filter:
-
-.. code-block:: sql
-
-    ALTER TABLE test_partitioned_table EXECUTE optimize
-    WHERE partition_key = 1
-
-expire_snapshots
-""""""""""""""""
-
-The ``expire_snapshots`` command removes all snapshots and all related metadata and data files.
-Regularly expiring snapshots is recommended to delete data files that are no longer needed,
-and to keep the size of table metadata small.
-The procedure affects all snapshots that are older than the time period configured with the ``retention_threshold`` parameter.
-
-``expire_snapshots`` can be run as follows:
-
-.. code-block:: sql
-
-  ALTER TABLE test_table EXECUTE expire_snapshots(retention_threshold => '7d')
-
-The value for ``retention_threshold`` must be higher than or equal to ``iceberg.expire_snapshots.min-retention`` in the catalog
-otherwise the procedure will fail with similar message:
-``Retention specified (1.00d) is shorter than the minimum retention configured in the system (7.00d)``.
-The default value for this property is ``7d``.
-
-remove_orphan_files
-"""""""""""""""""""
-
-The ``remove_orphan_files`` command removes all files from table's data directory which are
-not linked from metadata files and that are older than the value of ``retention_threshold`` parameter.
-Deleting orphan files from time to time is recommended to keep size of table's data directory under control.
-
-``remove_orphan_files`` can be run as follows:
-
-.. code-block:: sql
-
-  ALTER TABLE test_table EXECUTE remove_orphan_files(retention_threshold => '7d')
-
-The value for ``retention_threshold`` must be higher than or equal to ``iceberg.remove_orphan_files.min-retention`` in the catalog
-otherwise the procedure will fail with similar message:
-``Retention specified (1.00d) is shorter than the minimum retention configured in the system (7.00d)``.
-The default value for this property is ``7d``.
-
-.. _drop-extended-stats:
-
-drop_extended_stats
-~~~~~~~~~~~~~~~~~~~
-
-The ``drop_extended_stats`` command removes all extended statistics information from
-the table.
-
-``drop_extended_stats`` can be run as follows:
-
-.. code-block:: sql
-
-  ALTER TABLE test_table EXECUTE drop_extended_stats
-
-.. _iceberg-alter-table-set-properties:
-
-ALTER TABLE SET PROPERTIES
-~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-The connector supports modifying the properties on existing tables using
-:ref:`ALTER TABLE SET PROPERTIES <alter-table-set-properties>`.
+.. _iceberg-hive-catalog:
 
-The following table properties can be updated after a table is created:
+Hive metastore catalog
+^^^^^^^^^^^^^^^^^^^^^^
 
-* ``format``
-* ``format_version``
-* ``partitioning``
-* ``sorted_by``
+The Hive metastore catalog is the default implementation.
```

Comment:
```suggestion
The Hive metastore catalog is the default.
```

Reply:
This is existing technical content and outside the scope of the PR.